whylog_vim/gui/windows.py

```diff
@@ -69,7 +69,15 @@ def __init__(self):
         self.windows = dict()
 
     def create_window(self, window_type, content, modifiable=False, size=None):
+        origin_filename = VimUIWrapper.get_current_filename()
+        self.origin_offset = VimUIWrapper.get_cursor_offset()
         self.windows[window_type] = Window(window_type, content, modifiable, size)
+        for window in six.iterkeys(self.windows):
+            if origin_filename.endswith(window):
```

Comment:
please add a comment which explains how this typically looks like (what are the values of variables)